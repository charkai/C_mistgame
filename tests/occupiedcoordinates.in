place A1
place A1
place A1
place B16
place B16
place A1
term
