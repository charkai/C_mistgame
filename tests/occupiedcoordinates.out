Occupied coordinate
Occupied coordinate
Occupied coordinate
Occupied coordinate
